integration-cli/docker_cli_run_unix_test.go

```diff
@@ -338,3 +338,17 @@ func (s *DockerSuite) TestStopContainerSignal(c *check.C) {
 		c.Fatalf("Expected `exit trapped` in the log, got %v", out)
 	}
 }
+
+func (s *DockerSuite) TestRunSwapLessThanMemoryLimit(c *check.C) {
+	testRequires(c, memoryLimitSupport)
+	testRequires(c, swapMemorySupport)
+	out, exitCode, err := dockerCmdWithError("run", "-m", "16m", "--memory-swap", "15m", "busybox", "echo", "test")
+
+	if err == nil || exitCode == 0 {
+		c.Fatalf("Should not get right information: %s, %v", out, err)
+	}
+
+	if !strings.Contains(out, "Minimum memoryswap limit should be larger than memory limit") {
+		c.Fatal("couldn't find the invalid value warning")
```

Comment:
something like `c.Fatalf("Expected %q, got %q", expected, out)` is better when tests fail and you want to see the output